File: frontend/src/app/dtos/product.dto.ts
export interface Product {
  id: number;
  name: string;

  expiration: Date;
  quantity: number;
  price: number;

  brl: number;
  cad: number;
  eur: number;
  mxn: number;
  gbp: number;

  isDeleted: boolean;
  createdAt: Date;
  updatedAt: Date;
}

File: backend/src/Products/Application/Dtos/ProductDto.cs
namespace Products.Application.Dtos
{
  public record ProductDto
    (
    int Id,
    string Name,
    int Quantity,
    decimal Price,
    DateTime Expiration,
    decimal BRL,
    decimal EUR,
    decimal CAD,
    decimal MXN,
    decimal GBP,
    bool IsDeleted,
    DateTime CreatedAt,
    DateTime? UpdatedAt
  );
}
